.github/workflows/lint-docs.yml

```diff
@@ -1,19 +1,19 @@
-name: Lint docs
+# name: Lint docs
```

Comment:
Can this be brought back in now?

Reply:
Ah my bad, i commented out every action but the one I was working and forgot abt this